src/Microsoft.Health.Fhir.Core/Features/Persistence/RawResource.cs

```diff
@@ -26,10 +28,16 @@ protected RawResource()
         }
 
         [JsonProperty("data")]
-        public string Data { get; protected set; }
+        public string Data { get; set; }
 
         [JsonProperty("format")]
         [JsonConverter(typeof(StringEnumConverter))]
         public FhirResourceFormat Format { get; protected set; }
+
+        [JsonProperty("versionSet")]
+        public bool VersionSet { get; protected set; }
+
+        [JsonProperty("lastUpdatedSet")]
+        public bool LastUpdatedSet { get; protected set; }
```

Comment:
LastUpdated should already be set

Reply:
Isn't it cleared out here: https://github.com/microsoft/fhir-server/blob/6474001b582994e703506d94dbb89c80c8b82c24/src/Microsoft.Health.Fhir.Shared.Core/Features/Persistence/RawResourceFactory.cs#L49  ?

Reply:
I see, so for earlier documents we will need to patch the value. Going forward, I don't think there is a reason this field should every be left out, since the value is always computed by the FHIR server. We should document that for for backcompat, we need to check this field too. 
We could also just have one flag for both properties, which would slightly reduce storage size.

Reply:
Updated so LastUpdated doesn't get cleared, and reduced to one flag.